### ISSUE TITLE
$XDG_CONFIG_HOME don't work.

### ISSUE THREAD

**Issue by ebc-2in2crc** [ebc-2in2crc/dt#23]
**Describe the bug**
$XDG_CONFIG_HOME don't work

**To Reproduce**

```
$ echo $XDG_CONFIG_HOME
/Users/myhomedir/.myconfig

$ cat $XDG_CONFIG_HOME/dt/.dt
yearonly = 2006

$ dt -o yearonly now
yearonly
```

**Expected behavior**

```
$ dt -o yearonly now
2018
```

**Screenshots**
none.

**Desktop (please complete the following information):**
 - OS: MacOS 10.13.6

**Comment by ebc-2in2crc** [comment]
Fixed by #25

**Comment by ebc-2in2crc** [comment]
Fixed by #26
